src/views/ui-components/itemCard.jsx

```diff
@@ -13,7 +13,6 @@ import {
 const ItemCard = data => {
     var string = "this\n\n\n\n\n is a multi\ line\ string"; 
```

Comment:
Probably safe to delete this variable, 'string', doesn't look like it is in use.